src/App/components/Preview.tsx

```diff
@@ -113,6 +114,27 @@ const PreviewVerse = (prop: { verse: string; highlightVerse: boolean; highlightC
   })}</>;
 };
 
+interface BackgroundSettings {
+  file: string;
+  color: string;
+  type: string;
+}
+
+const getViewBlob = (background: BackgroundSettings): string => {
+  if (!background.file) {
+    return "";
+  }
+  const URL = window.URL || window.webkitURL;
+  let video = null;
+  try {
+    video = fs.readFileSync(background.file);
+    const fileURL = URL.createObjectURL(new Blob([video], { "type": "video/mp4" }));
```

Comment:
The preview doesn't seem to work for image backgrounds. I think you need to account for that here?